src/api/options-lifecycle.md

```diff
@@ -209,7 +209,7 @@ Called when an error propagating from a descendant component has been captured.
 
   - An `errorCaptured` hook can return `false` to prevent the error from propagating further. This is essentially saying "this error has been handled and should be ignored." It will prevent any additional `errorCaptured` hooks or `app.config.errorHandler` from being invoked for this error.
 
-## renderTracked <sup class="vt-badge dev-only" />
+## renderTracked <sup class="vt-badge dev-only" /> {#rendertracked-sup-class-vt-badge-dev-only}
```

Comment:
The original hash names like this one on the website (`#rendertracked`) actually doesn't include the `-sup-*` things.

https://vuejs.org/api/options-lifecycle.html#rendertracked

I think we'd better still follow this rule to avoid breaking changes.

Thanks.

Reply:
fixed